### PR TITLE
Nukes any mention of CollisionGroup.Clickable

### LINKED ISSUE #2181: CollisionGroup.Clickable is deprecated and should be removed

### PR DESCRIPTION
**Pull request by SweptWasTaken**
## About the PR 

Just removes it from everything that had it.

Closes  #2181.

### CODE REVIEW

File: Resources/Prototypes/Entities/Constructible/Specific/traitordm.yml [hunk 1]
```diff
@@ -18,10 +18,6 @@
     - shape:
         !type:PhysShapeAabb
           bounds: "-0.25,-0.25,0.25,0.25"
-      mass: 1
```
Comment (mirrorcult) [inline Resources/Prototypes/Entities/Constructible/Specific/traitordm.yml:21]: intended?

File: Content.Shared/Physics/CollisionGroup.cs [hunk 2]
```diff
@@ -21,10 +21,9 @@ public enum CollisionGroup
 		MobImpassable   = 1 <<  2, // 4 Mobs, players, crabs, etc
 		VaultImpassable = 1 <<  3, // 8 Things that cannot be jumped over, not half walls or tables
 		SmallImpassable = 1 <<  4, // 16 Things a smaller object - a cat, a crab - can't go through - a wall, but not a computer terminal or a table
-        Clickable       = 1 <<  5, // 32 Temporary "dummy" layer to ensure that objects can still be clicked even if they don't collide with anything (you can't interact with objects that have no layer, including items)
-        GhostImpassable = 1 <<  6, // 64 Things impassible by ghosts/observers, ie blessed tiles or forcefields
-        Underplating    = 1 <<  7, // 128 Things that are under plating
-        Passable        = 1 <<  8, // 256 Things that are passable
+        GhostImpassable = 1 <<  5, // 64 Things impassible by ghosts/observers, ie blessed tiles or forcefields
+        Underplating    = 1 <<  6, // 128 Things that are under plating
+        Passable        = 1 <<  7, // 256 Things that are passable
```
Comment (metalgearsloth) [inline Content.Shared/Physics/CollisionGroup.cs:26]: You didn't update the integer references

**Review by metalgearsloth — CHANGES_REQUESTED** [review]
Stuff with no mask or layer shouldn't exist so those physics components will need removing.

**Comment by SweptWasTaken** [comment]
Reviews applied, I moved stuff like the chargers and the morgue stuff over to Constructible while I was at it.

**Review by metalgearsloth — APPROVED** [review]

**Pull request merged**